kedro-datasets/pyproject.toml

```diff
@@ -248,8 +259,7 @@ test = [
     "scipy>=1.7.3",
     "packaging",
     "SQLAlchemy>=1.2",
-    "tables>=3.8.0; platform_system == 'Windows'",  # Import issues with python 3.8 with pytables pinning to 3.8.0 fixes this https://github.com/PyTables/PyTables/issues/933#issuecomment-1555917593
-    "tables~=3.6; platform_system != 'Windows'",
+    "tables~=3.6",
```

Comment:
Does this work with a higher pin?

Reply:
Yeah please, would be good to try `tables>=3.6`